From: Bernhard Rosenkränzer <bero@linaro.org>
Date: Fri, 30 Jun 2017 15:39:01 +0000 (+0200)
Subject: Fix detection of AT-SPI
X-Git-Url: https://codereview.qt-project.org/gitweb?p=qt%2Fqtbase.git;a=commitdiff_plain;h=386af91c57948b88ba98cb040430e4a3878e3e3b;hp=989b9da3007ff06c79ab7ee553acfce9177df5b6

Fix detection of AT-SPI

Building qtbase 5.9.1 on Linux always results in a build with
accessibility support disabled.

The problem is that the config option for accessibility-atspi-bridge
checks for config.atspi, which isn't defined anywhere - it should
check for libs.atspi (which is set if pkg-config finds atspi-2) instead.

[ChangeLog][QtGui][Platform Specific Changes][Linux/XCB]
Fixed detection of AT-SPI, allowing accessibility support to be built
again.

Task-number: QTBUG-61731
Change-Id: If3bd5dfccda40158c566f8507e34b6877b59b6fb
Reviewed-by: Frederik Gladhorn <frederik.gladhorn@qt.io>
Reviewed-by: Bernhard Rosenkränzer <bero@lindev.ch>
---

diff --git a/src/gui/configure.json b/src/gui/configure.json
index a2a78aa..ef7ba51 100644
--- a/src/gui/configure.json
+++ b/src/gui/configure.json
@@ -434,7 +434,7 @@
     "features": {
         "accessibility-atspi-bridge": {
             "label": "ATSPI Bridge",
-            "condition": "features.accessibility && features.xcb && features.dbus && config.atspi",
+            "condition": "features.accessibility && features.xcb && features.dbus && libs.atspi",
             "output": [ "privateFeature", "feature" ]
         },
         "angle": {
